sdks/wasm/src/driver.c

```diff
@@ -47,6 +47,10 @@ static MonoClass* datetimeoffset_class;
 
 int mono_wasm_enable_gc;
 
+MONO_API void*  xamarin_timezone_get_data (MonoString* name, int *size);
+MONO_API char**  xamarin_timezone_get_names (int *count);
+MONO_API MonoString*  xamarin_timezone_get_local_name (void);
```

Comment:
Is `xamarin_` a new prefix, vs. `mono_`?
Put in a header?

Reply:
These are all `mono_` now

Reply:
Its ok to name them xamarin, mono is just the prefix for the runtime. Its ok to name them mono as well.